### DISCUSSION TITLE
About Vercel website

### DISCUSSION THREAD

**Discussion by livc** [vercel/vercel#5242] [Help] [answered]
Hi, I want to know if I can find Vercel website source code on github? I think it's well-designed and I want to learn its source code.

I found https://github.com/vercel/next-site in github but no Vercel site.

Thanks.

**Comment by styfle** [comment, marked as answer]
Hi @livc 

The website is not open source at this time.

We might do so in the future.

We'll probably start first with components and then eventually the whole website.
